### DISCUSSION TITLE
CELEBRATIONS!

### DISCUSSION THREAD

**Discussion by N3RDIUM** [N3RDIUM/zen#13] [Announcements]
![image](https://user-images.githubusercontent.com/74598401/135040774-caf95e55-b70e-4b78-9909-94fb91a0ea98.png)

## 🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉

**Comment by Super45coder** [comment]
Yeah, I am in school in the week as well.
________________________________
From: somePythonProgrammer ***@***.***>
Sent: Saturday, October 2, 2021 4:39 AM
To: somePythonProgrammer/JARVIS ***@***.***>
Cc: Goradia, Aarush H ***@***.***>; Mention ***@***.***>
Subject: [somePythonProgrammer/JARVIS] CELEBRATIONS! (Discussion #13)

CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.


[image]<https://user-images.githubusercontent.com/74598401/135040774-caf95e55-b70e-4b78-9909-94fb91a0ea98.png>

We have made ANOTHER release, WITH an installer!
Now JARVIS can run on (Windows) devices without python!
Credit to @Super45coder<https://github.com/Super45coder> @Sreehari521<https://github.com/Sreehari521> @e-coders<https://github.com/e-coders> ( Genius398 [BUSY] ) !
🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/somePythonProgrammer/JARVIS/discussions/13>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATDMIRSEXEWICTUHMHTIYEDUE3OOHANCNFSM5FGIG54A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
